src/main/java/edu/wright/airviewer2/AIRViewerController.java

```diff
@@ -92,7 +102,13 @@ private AIRViewerModel promptLoadModel(String startPath) {
             File file = fileChooser.showOpenDialog(stage);
             if (null != file) {
                 String path = file.getCanonicalPath();
+                while(!path.endsWith(".pdf")) {
+                	System.out.println("select only pdf format files");
+                	file = fileChooser.showOpenDialog(stage);
+                	path = file.getCanonicalPath();
+                }
```

Comment:
Can you take the latest from master every time when you create PR. I see this issue is already merged to master.

Reply:
Sync done with the latest changes